packages/balancer/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { Chains } from './utils'
+
+export const VAULT_CONTRACT = '0xBA12222222228d8Ba445958a75a0704d566BF2C8'
```

Comment:
Is this the right case?

Reply:
I'll change it to lowercase since we will be using the $or array.